### DISCUSSION TITLE
Use in company with separate users and different rights?

### DISCUSSION THREAD

**Discussion by schoettl** [jonaswinkler/paperless-ng#1626] [Support] [answered]
I wonder if paperless-ng has a solution for this use-case:

In our company we're considering using paperless. But the clerk user, for example, should only see invoices but no letters from the lawyer.

**Comment by azapater** [comment, marked as answer]
The most voted topic is about this: 

https://github.com/jonaswinkler/paperless-ng/discussions/52

**Comment by schoettl** [comment]
Thanks!
